package Day61_ErrorHandling_Exeptions;

public class Fatih_HataBulma2 {
    public static void main(String[] args) {
        // Step 1 : exception alma
//        System.out.println("Hi class ");
//
//        String a=null;
//
//       // System.out.println(a.substring(0,1));
//
//        System.out.println("By class"); // öncesinde exception olduğu için consolda yazılmadı


        //Step 2 : Try Catch

//        System.out.println("Hi class- Before exception");
//
//        try {
//
//            System.out.println(1/(1-1));  // exception beklediğimiz kodu try bloke içine yazdık
//        }catch (Exception e){   //=>  beklenen muhtemel exception olarak ilgili class ismi ile bir parametre girdik
//            System.out.println("Sıfıra Bölündü, TEST FAILED !!"); // exception olması durumunda istediğimiz mesajı yazdık
//        }
//        System.out.println("By class- After exception");


    }

}

